src/scss/main.scss

```diff
@@ -96,12 +115,78 @@ input[type="email"] {
   }
 }
 
+//* How it works *//
+.how-it-works {
+  img {
+    width: 12rem;
+  }
+
+  h4 {
+    color: var(--orange-400);
+  }
+}
+
+//* Money back banner *//
+.main-container:has(.money-back) {
+ margin-top: 4rem;
+}
+
+//* Why BeakBox *//
+.why-beakbox {
+  --container-width: 95ch;
+  
+  h2 {
+    margin-bottom: 5rem;
+    text-align: center;
+  }
+
+}
+
+.why-beakbox:has(.flex-container-collapse) {
```

Comment:
`:has` is pretty new, will the design be okay if this rule (and its children) is ignored and doesn't take effect? If it's just a little less pretty but totally readable, that's fine - good progressive enhancement :)

Reply:
If I reduce the original image size, it is going to look OK. Atm, the image takes 90% of the section, the text is still readable though (not very pretty).

Reply:
I looked at this a little more closely. You are only using `.why-beakbox` in one place. That element has a `.flex-container-collapse` in it. So there seems to never be a time when `:has(.flex-container-collapse)` would be false, rendering this selector unnecessarily complex. Just do `.why-beakbox` with the nested rules underneath it.